nixos/modules/config/xdg/portals/wlr.nix

```diff
@@ -0,0 +1,69 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.xdg.portal.wlr;
+  settingsFormat = pkgs.formats.ini { };
+  configFile = settingsFormat.generate "xdg-desktop-portal-wlr.ini" cfg.settings;
+in
+{
+  meta = {
+    maintainers = with maintainers; [ minijackson ];
+  };
+
+  options.xdg.portal.wlr = {
+    enable = mkEnableOption ''
+      desktop portal for wlroots-based desktops
+
+      This will add the <package>xdg-desktop-portal-wlr</package> package into
+      the <option>xdg.portal.extraPortals</option> option, and provide the
+      configuration file
+    '';
+
+    package = mkOption {
+      description = "Package containing the wlr desktop portal";
+      type = types.package;
+      default = pkgs.xdg-desktop-portal-wlr;
+    };
+
+    settings = mkOption {
+      description = ''
+        Configuration for <package>xdg-desktop-portal-wlr</package>.
+
+        See <command>man 5 xdg-desktop-portal-wlr</command> for supported
+        values.
+      '';
+
+      type = types.submodule {
+        freeformType = settingsFormat.type;
+      };
+
+      default = { };
+
+      # Example taken from the manpage
+      example = literalExample ''
+        {
+          screencast = {
+            output_name = "HDMI-A-1";
+            max_fps = 30;
+            exec_before = "disable_notifications.sh";
+            exec_after = "enable_notifications.sh";
+            chooser_type = "simple";
+            chooser_cmd = "${pkgs.slurp}/bin/slurp -f %o -or";
```

Comment:
The variable should probably be escaped.

Reply:
I actually escaped it before, but for some reason when unescaped it shows up properly in the manpage and manual. I'm not sure why...

Reply:
Turns out it is being scrubbed here: https://github.com/NixOS/nixpkgs/blob/17bcbee9fa5aabb3e0b4015b71eb6578da2d6fe3/nixos/modules/misc/documentation.nix#L21-L40

Escaping it would be still preferable since the scrubbing does not happen when browsing options in `nix repl`.